product_template_tags/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Product Template Tags',
+    'summary': """
+        This addon allow to add tags on products""",
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV, Odoo Community Association (OCA), Numigi',
+    'website': 'https://www.acsone.eu',
```

Comment:
URL should be `https://github.com/OCA/product-attribute`

Reply:
Change has been made, thanks.